static/gsAdmin/components/planList.tsx

```diff
@@ -179,14 +177,15 @@ function PlanList({
         <StyledFormSection>
           <h4>Available Products</h4>
           {availableProducts.map(productInfo => {
+            const addOnKey = `addOn${toTitleCase(productInfo.productName)}`;
```

Comment:
i forgot how `toTitleCase` works (ie. if it forces lower casing) but it might be better to just use `upperFirst` or `capitalize` on `apiName` since `apiName` should already be in camel case

Reply:
but ig this won't be a problem til we have a product with multiple words